string message
float32 axis1
float32 axis2
float32 axis3
float32 axis4
int32 b0
int32 b1
int32 b2
int32 b3
int32 b4
int32 b5
int32 b6
int32 b7
int32 b8
int32 b9
int32 b10
int32 b11
